wear/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomNavigator.kt

```diff
@@ -0,0 +1,36 @@
+package chat.rocket.android.chatroom.presentation
+
+import chat.rocket.android.R
+import chat.rocket.android.chatroom.models.messages.MessageUiModel
+import chat.rocket.android.chatroom.ui.ChatRoomActivity
+import chat.rocket.android.chatroom.ui.newInstance
+import chat.rocket.android.util.addFragmentBackStack
+import chat.rocket.android.util.removeFragmentBackStack
+
+class ChatRoomNavigator(internal val activity: ChatRoomActivity) {
```

Comment:
Prefer to define constants on `ChatRoomFragment`, `CompleteChatRoomFragment` and `ReplyMessage` with the tag name. 

Some days ago I needed to replace all this way that we had done by putting the tag constants on all the fragments because I need to use the fragment names (the tags) with our analytics tool.